composer.json

```
@@ -3,12 +3,12 @@
     "description":  "REST Library for the Discord API",
     "type":         "library",
     "require":      {
-        "php":                        "^5.6|^7.0",
+        "php":                        "^5.6|^7.0|^8.0",
```

Lets fully shore up to minimum PHP EOL versions. Might as well

```suggestion
        "php":                        "^7.3|^8.0",
```